java/compaction-event-store/src/test/java/sleeper/compaction/dynamodb/DynamoDBCompactionJobEventStoreCreatorIT.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.compaction.dynamodb;
+
+import com.amazonaws.services.dynamodbv2.model.DescribeTableResult;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static sleeper.compaction.dynamodb.DynamoDBCompactionJobEventStore.jobEventsTableName;
+
+public class DynamoDBCompactionJobEventStoreCreatorIT extends DynamoDBTestBase {
+
+    @Test
+    public void shouldCreateStore() {
+        String instanceId = "test-instance";
+        DynamoDBCompactionJobEventStoreCreator creator = new DynamoDBCompactionJobEventStoreCreator(instanceId, dynamoDBClient);
+
+        creator.create();
+
+        assertThat(dynamoDBClient.describeTable(jobEventsTableName(instanceId)))
+                .extracting(DescribeTableResult::getTable).isNotNull();
+    }
```

Comment:
Can we try to stick to the given/when/then format for tests?

Reply:
I've added given/when/then comments.